polyglot/piranha/src/models/unit_tests/scopes_test.rs

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
Why these empty lines? Also, um... now that I think about this, aren't a bunch of our files missing a copyright/license header (not for changing in this PR, but...)?

Reply:
will add copyright header everywhere missing.

Reply:
I will make that a separate PR.